### ISSUE TITLE
nan loss although multiple outputs loss not nan

### ISSUE THREAD

**Issue by Avi-avidan** [keras-team/keras#11014]
Using model with multiple outputs I am getting nan loss despite all outputs loss seems valid. screenshot attached.
This behavior remains when I change loss functions (I have tried standard 'mae', 'mse', 'categorical_crossentropy' and some customized loss functions as well).
This behavior remains when I change optimizers (I have tries Adam and SGD with momentum).

Any advice would be appreciated.

multiple loss do seem to converge though :-| (screenshot attached).
![image](https://user-images.githubusercontent.com/25052915/44705551-a2a85000-aaa7-11e8-9b94-be5a7b3f0311.png)

![image](https://user-images.githubusercontent.com/25052915/44705587-bb186a80-aaa7-11e8-818b-7f7a4478c5cc.png)

loss & val loss are nan -
![image](https://user-images.githubusercontent.com/25052915/44705694-06cb1400-aaa8-11e8-8a43-5ee17f353eed.png)

**Comment by timodonnell** [comment]
I'm hitting this now. Did you ever sort this out?

**Comment by soroush361** [comment]
I'm not sure if it resolves your issue or not, but try to change your input data type to 'float64'.
`x_train = x_train.astype('float64')`
It worked in my case.
